core/domain/suggestion_registry.py

```diff
@@ -575,10 +575,17 @@ def validate(self):
                     suggestion_models.SCORE_TYPE_TRANSLATION,
                     self.get_score_type()))
 
-        if self.change.cmd != exp_domain.CMD_ADD_TRANSLATION:
+        # Suggestions in the future will only use CMD_ADD_WRITTEN_TRANSLATION.
+        # CMD_ADD_TRANSLATION is added in the following check to support older
+        # suggestions.
+        accepted_cmds = [
+            exp_domain.CMD_ADD_TRANSLATION,
```

Comment:
OR maybe change the variable name of this command: `DEPRECATED_CMD_ADD_TRANSLATION`